packages/v4/src/content/design-guidelines/content/capitalization.md

```diff
@@ -5,29 +5,78 @@ hideTOC: true
+Consistent capitalization adds clarity and consistency across your products' UIs.
 
-Sentence case capitalizes only the first word of each sentence and proper nouns. On PatternFly, we use sentence case capitalization for all text elements, except the following:
-- Product names (Red Hat Virtualization, PatternFly, Red Hat CloudForms)
-- Proper nouns for individuals, organizations, and locations.
-- Acronyms and initialisms (URL, HTML, CSS, JS)
+## Capitalization types
 
-**Note**: Hyperlinked articles in a UI are usually not displayed as titles. Instead, the copy typically reads something along the lines of, “Learn more about [tool name].”
+* **Title case**: Title case capitalizes the first letter of each word in a sentence except for smaller words like articles and conjunctions that have fewer than four letters. 
 
-## Capitalization for tools outside your product portfolio
-If you’re referencing tools outside your company’s product portfolio, write the product names as they appear in the respective company’s documentation. For example, if you’re referencing a product in AWS and Amazon capitalizes it, then capitalize it in your writing.
+    <br />Example: "PatternFly Design Resources With Tips and Best Practices"
+    <br />Hyperlinked articles in a UI are not displayed as titles. Instead, the copy typically reads something like, "Learn more about [tool name]."
+<br />
+* **Sentence case**: Sentence case only capitalizes the first letter of the first word in a title, heading, subtitle, or subheading except for proper nouns, product names, acronyms, and initialisms.
+    <br />Example: “PatternFly’s design resources with tips and best practices”
 
-## Capitalization for breadcrumb trails
-Usually, page titles appear in a breadcrumb trail. If all page titles are in sentence case, they should also appear in sentence case in the breadcrumb trail.
+## Capitalization across PatternFly
+Follow these guidelines for **PatternFly content**, including documentation, UX copy, blog articles, and more. 
 
-However, breadcrumb trails aren’t limited to page titles alone. If a name entered by a user (such as “vagrant-host”) is included in the breadcrumb trail, then use the same case that the name appears in.
+* Use **sentence case** for page titles, menu/navigation items, headings, subtitles, and subheadings across PatternFly. Capitalize proper nouns, product names, acronyms, and initialisms.  
+    <br />Example: “Introduction to HTML and table components”  
+<br />
+
+* Write all components in **lowercase** unless they start a sentence. 
+    <br />Example: "Card, button, and banner components are my favorites."  
+    <br />**Note**: “React” is always capitalized, even when referring to a component (“React component”). It’s the official name of a JavaScript library, and it’s written as a proper noun across the [react.js org site](https://reactjs.org/).
```

Comment:
Not sure why, but the "Note" doesn't show up bolded, and instead shows the stars.
![image](https://user-images.githubusercontent.com/51165119/89815754-4c593e00-db13-11ea-868a-3c447e8cbb2d.png)

Reply:
Might have something to do with the <br /> before it... I'll tinker with it and see